torch/fx/experimental/symbolic_shapes.py

```diff
@@ -3267,6 +3258,10 @@ def _maybe_evaluate_static(
             ):
                 new_range_env[k] = vr
                 continue
+            if size_oblivious and k in self.size_like:
+                lower = max(2, vr.lower)
+            else:
+                lower = vr.lower
```

Comment:
This should probably go before the previous if, as this gives a non trivial lower bound.